SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,55 +1,70 @@
 <%(TOC)%>
 # Singleton Actors
 
-Singleton Actors allow a single source of truth for both operations and data across a multiserver simulation. They are server-side authoritative [Unreal Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS. For example, if you are implementing a scoreboard, you'd most likely only want there to be one of them in your world. Ensuring this behavior in a multiserver paradigm requires a few additional steps which can be easily facilitated through the Singleton Actor.
+You can use a Singleton Actor to define a single source of truth for operations or data, or both, across a game world that uses zoning. You can have only one instance of an [entity]({{urlRoot}}/content/glossary#entity) that represents a Singleton Actor per game world.
 
-There are two kinds of Singleton Actors:
+You define a Singleton Actor by tagging an Actor with the `SpatialType=Singleton` class attribute. For example, if you are implementing a scoreboard, you probably want only one scoreboard in your world, so you can tag the scoreboard Actor as a Singleton Actor.
 
-* **Public Singleton Actors** - Singleton Actors which are replicated to [server-workers and client-workers]({{urlRoot}}/content/glossary#workers). [AGameState](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Public Singleton Actor.
-* **Private Singleton Actors** - Singleton Actors which are replicated to [server-workers]({{urlRoot}}/content/glossary#workers), but not accessible to [client-workers]({{urlRoot}}/content/glossary#workers). [AGameMode](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Private Singleton Actor.
+There are two types of Singleton Actors:
 
-You can define any class as a Singleton Actor. Unreal engine classes we have explicitaly tagged as Singleton Actors are -
+* **Public Singleton Actors** - Singleton Actors that are replicated to [server-workers and client-worker instances]({{urlRoot}}/content/glossary#workers). [`AGameState`](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) and `ALevelScriptActor` are Public Singleton Actors.
+* **Private Singleton Actors** - Singleton Actors that are replicated to [server-worker instances]({{urlRoot}}/content/glossary#workers), but not accessible to [client-worker instances]({{urlRoot}}/content/glossary#workers). [`AGameMode`](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Private Singleton Actor.
 
-1. AGameModeBase
-1. AGameStateBase
+You can define any class as a Singleton Actor. However, the following Unreal Engine classes are automatically tagged in source code as Singleton Actors, so you don’t need to define them:
 
-* As `SpatialType` is inheritable, all classes that derive off these classes are also considered Singleton Actors. You can opt out using the `NotSpatialType` tag.
+* `AGameModeBase`
+* `AGameStateBase`
+* `ALevelScriptActor`
```

Comment:
Do we need a link for this one? I found https://api.unrealengine.com/INT/API/Runtime/Engine/Engine/ALevelScriptActor/index.html but I'm not sure if that's appropriate. It's not the same type of link as https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode.

Reply:
Per Mike's reply, we can add a link to that though this link is not part of docs...